taipy/gui_core/_GuiCoreLib.py

```diff
@@ -12,18 +12,27 @@
 import typing as t
 from datetime import datetime
 
+from taipy.core import Cycle, DataNode, Job, Scenario, Sequence, Task
 from taipy.gui import Gui
 from taipy.gui.extension import Element, ElementLibrary, ElementProperty, PropertyType
 
 from ..version import _get_version
 from ._adapters import (
+    _GCDoNotUpdate,
```

Comment:
Not too happy with this symbol name... compared to the others.
`_GuiCoreNoUpdate`?